Algorithm.CSharp/OpeningBreakoutAlgorithm.cs

```diff
@@ -288,7 +288,7 @@ private void ScanForEntrance()
 
             var allowedDollarLoss = MaximumPorfolioRiskPercentPerPosition * Portfolio.TotalPortfolioValue;
 
-            var shares = (int) (allowedDollarLoss/expectedCaptureRange);
+            var shares = allowedDollarLoss/expectedCaptureRange;
```

Comment:
Think this is going to need to happen. Will also need:

`SetHoldings<decimal>(Symbol, percentage)`

Too keep consistency the non generic will return a floored quantity but the new generics will return a fractional quantity. Makes sense?